lnx/action.yml

```diff
@@ -0,0 +1,90 @@
+name: Deploy Setup Action for Linux/OSX
+description: Download direct tools
+inputs:
+  solsta_client_id:
+    description: Client ID to authenticate usage of Solid State Networks console tools
+    required: true
+    default: ""
+  solsta_client_secret:
+    description: Secret Key to authenticate usage of Solid State Networks console tools
+    required: true
+    default: ""
+  console_version:
+    description: Version of Solsta Console Tools to use
+    required: true
+    default: "6.1.2.51"
+  scripts_version:
+    description: Version of Solsta Deploy Scripts to use
+    required: true
+    default: "3.7.18"
+
+outputs:
+  result:
+    description: The state of the action, if processes completed successfully
+
+runs:
+  using: composite
+  steps:
+    - name: Install system dependencies
+      shell: bash
```

Comment:
Do you need `shell` for all the steps? I think bash is the default on linux/mac and pwsh is the default on windows? @nmoinvaz - is that right?

Reply:
I don't think we need the special version of the YAML that requires powershell. You can use bash shell on Windows too.

Reply:
In other words, you would only need it if it was being run on Windows too.

Reply:
I emailed @bozimmerman last night to tell him how to do it.

Reply:
Using bash shell on Windows:
1. apt-get failed.
2. I removed apt-get, and then wget failed for being not-found
I can keep removing things to uncover every failure, but money says the 7zip would fail next, followed by the python path (which I can maybe fix), and finally the "if [ ! -d ..." would fail, which is what I saw when playing with the deploy action.)

Reply:
mkdir and rm worked fine, but then, those work like that in powershell also.
Are you sure shell:bash in windows isn't just running powershell?

Reply:
I believe `shell: bash` in Windows uses the Git bash.

Reply:
@bozimmerman to run tasks conditionally on Windows in the same workflow do:
```
    - name: Install packages (Windows)
      if: runner.os == 'Windows'
      run: choco install cmake
```

Reply:
OK, so I've now consolidated the Windows/Linux stuff, used bash everywhere, etc.